docs/operator/add-custom-scrape-jobs.md

```diff
@@ -121,3 +121,9 @@ spec:
 ```
 
 The Secret **MUST** be in the same namespace as the PrometheusInstance.
+
+There is a known [issue](https://github.com/grafana/agent/issues/655) that
+currently prevents the Grafana Agent Operator from updating Grafana Agent
+deployments when `additionalScrapeConfigs` or the underlying secret changes.
+Until the issue is resolved, you should restart the Operator to force it to pick
```

Comment:
Does it make sense to add an appropriate command? I.E. `kubectl rollout restart deployment agent-operator`, or similar?

Reply:
Good point. It's tricky because we don't prescribe the name of the deployment, but we can assume they followed the getting started guide and use that. Updating

Reply:
PTAL :)